From 1f9256b22acbd17b9e703045b9aaa880f7b38fdb Mon Sep 17 00:00:00 2001
From: maxice8 <thinkabit.ukim@gmail.com>
Date: Mon, 11 Jul 2016 17:25:00 -0300
Subject: [PATCH] Fix backtrace detection on musl


diff --git a/Source/WTF/wtf/Assertions.cpp b/Source/WTF/wtf/Assertions.cpp
index 2354550..d5c47e1 100644
--- a/Source/WTF/wtf/Assertions.cpp
+++ b/Source/WTF/wtf/Assertions.cpp
@@ -66,7 +66,7 @@
 #if OS(DARWIN) || (OS(LINUX) && defined(__GLIBC__) && !defined(__UCLIBC__))
 #include <cxxabi.h>
 #include <dlfcn.h>
-#include <execinfo.h>
+//#include <execinfo.h>
 #endif
 
 extern "C" {
@@ -234,7 +234,7 @@ void WTFReportArgumentAssertionFailure(const char* file, int line, const char* f
 void WTFGetBacktrace(void** stack, int* size)
 {
 #if OS(DARWIN) || (OS(LINUX) && defined(__GLIBC__) && !defined(__UCLIBC__))
-    *size = backtrace(stack, *size);
+    // *size = backtrace(stack, *size);
 #elif OS(WINDOWS) && !OS(WINCE)
     // The CaptureStackBackTrace function is available in XP, but it is not defined
     // in the Windows Server 2003 R2 Platform SDK. So, we'll grab the function
@@ -280,9 +280,9 @@ void WTFReportBacktrace()
 void WTFPrintBacktrace(void** stack, int size)
 {
 #if USE(BACKTRACE_SYMBOLS)
-    char** symbols = backtrace_symbols(stack, size);
-    if (!symbols)
-        return;
+//    char** symbols = backtrace_symbols(stack, size);
+//    if (!symbols)
+//        return;
 #endif
 
     for (int i = 0; i < size; ++i) {
-- 
2.9.0.56.g674d38f.dirty

